### ISSUE TITLE
Manual changes of maintenance time in the portal breaks the terraform workflow

### ISSUE THREAD

**Issue by JanGrosse** [stackitcloud/terraform-provider-stackit#420]
Hello,

we have multiple SKE clusters which we manage through the terraform provider. In none of those did we define the `maintenance` object in the past as this field is optional. 
Now we changed the maintenance time window inside the STACKIT portal manually, afterwards the terraform provider complains about a mismatch and does not perform any plan / apply anymore.

```
│ Processing API payload: mapping maintenance: getting maintenance times: start time '0000-01-01T14:30:00Z' from API response doesn't match start time '05:48:00Z' from TF config
```

As mentioned we never actual defined the maintenance in terraform so I am not sure why terraform even has this value.
It did not help to set the maintenance object afterwards (I tried to) in terraform with the updated time, it did also not help to update the provider version (was running on 0.14 before, updated to 0.19). The only thing which worked was to manually edit the state in s3 to match the new time which was set in the portal itself.

Greetings,
JanGrosse.

**Comment by joaopalet** [comment]
Hi @JanGrosse,

This is indeed an issue from our side. We will have a look into it, thanks for opening the issue!

**Comment by joaopalet** [comment]
Hi @JanGrosse,

This was fixed in #444 and released in version [v0.23.0](https://registry.terraform.io/providers/stackitcloud/stackit/0.23.0)
